[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:1003 Starting new HTTPS connection (1): pypi.python.org:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:456 https://pypi.python.org:443 "GET /packages/source/s/sympy/sympy-1.10.tar.gz HTTP/1.1" 301 122
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:1003 Starting new HTTPS connection (1): pypi.org:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:456 https://pypi.org:443 "GET /packages/source/s/sympy/sympy-1.10.tar.gz HTTP/1.1" 301 269
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:456 https://files.pythonhosted.org:443 "GET /packages/source/s/sympy/sympy-1.10.tar.gz HTTP/1.1" 302 0
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:456 https://files.pythonhosted.org:443 "GET /packages/7d/23/70fa970c07f0960f7543af982d2554be805e1034b9dcee9cb3082ce80f80/sympy-1.10.tar.gz HTTP/1.1" 200 7110645
[35mDEBUG   [0m conda_index.index:__init__.py:680 found subdirs {'noarch', 'linux-64'}
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:101 CondaIndexCache channel_root=/home/runner/work/_temp/parallel/popen-gw0/test_pypi_url0, subdir=linux-64 db_filename=/home/runner/work/_temp/parallel/popen-gw0/test_pypi_url0/linux-64/.cache/cache.db cache_is_brand_new=True
[32mINFO    [0m conda_index.index.convert_cache:convert_cache.py:150 Migrate database
[32mINFO    [0m conda_index.index.convert_cache:convert_cache.py:199 CONVERT .cache
[32mINFO    [0m conda_index.index.convert_cache:convert_cache.py:150 Migrate database
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:431 linux-64 listdir
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:450 linux-64 save fs state
[35mDEBUG   [0m conda_index.index:__init__.py:759 linux-64 find packages to extract
[35mDEBUG   [0m conda_index.index:__init__.py:771 linux-64 extract 0 packages
[32mINFO    [0m conda_index.index:__init__.py:801 linux-64 cached 0 B from 0 packages at 0 B/second
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:101 CondaIndexCache channel_root=/home/runner/work/_temp/parallel/popen-gw0/test_pypi_url0, subdir=noarch db_filename=/home/runner/work/_temp/parallel/popen-gw0/test_pypi_url0/noarch/.cache/cache.db cache_is_brand_new=True
[32mINFO    [0m conda_index.index.convert_cache:convert_cache.py:150 Migrate database
[32mINFO    [0m conda_index.index.convert_cache:convert_cache.py:199 CONVERT .cache
[32mINFO    [0m conda_index.index.convert_cache:convert_cache.py:150 Migrate database
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:431 noarch listdir
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:450 noarch save fs state
[35mDEBUG   [0m conda_index.index:__init__.py:759 noarch find packages to extract
[35mDEBUG   [0m conda_index.index:__init__.py:771 noarch extract 0 packages
[32mINFO    [0m conda_index.index:__init__.py:801 noarch cached 0 B from 0 packages at 0 B/second
[32mINFO    [0m conda_index.index:__init__.py:578 Completed linux-64
[32mINFO    [0m conda_index.index:__init__.py:578 Completed noarch
[35mDEBUG   [0m conda_index.index:__init__.py:680 found subdirs {'noarch', 'linux-64'}
[32mINFO    [0m conda_index.index:__init__.py:653 Channeldata subdir: linux-64
[35mDEBUG   [0m conda_index.index:__init__.py:654 linux-64 read repodata
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:101 CondaIndexCache channel_root=/home/runner/work/_temp/parallel/popen-gw0/test_pypi_url0, subdir=linux-64 db_filename=/home/runner/work/_temp/parallel/popen-gw0/test_pypi_url0/linux-64/.cache/cache.db cache_is_brand_new=False
[35mDEBUG   [0m conda_index.index:__init__.py:662 linux-64 channeldata finished
[32mINFO    [0m conda_index.index:__init__.py:653 Channeldata subdir: noarch
[35mDEBUG   [0m conda_index.index:__init__.py:654 noarch read repodata
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:101 CondaIndexCache channel_root=/home/runner/work/_temp/parallel/popen-gw0/test_pypi_url0, subdir=noarch db_filename=/home/runner/work/_temp/parallel/popen-gw0/test_pypi_url0/noarch/.cache/cache.db cache_is_brand_new=False
[35mDEBUG   [0m conda_index.index:__init__.py:662 noarch channeldata finished
[35mDEBUG   [0m conda_index.index:__init__.py:1155 _maybe_write /home/runner/work/_temp/parallel/popen-gw0/test_pypi_url0/index.html to /home/runner/work/_temp/parallel/popen-gw0/test_pypi_url0/index.html
[35mDEBUG   [0m conda_index.index:__init__.py:670 write channeldata
[35mDEBUG   [0m conda_index.index:__init__.py:1155 _maybe_write /home/runner/work/_temp/parallel/popen-gw0/test_pypi_url0/channeldata.json to /home/runner/work/_temp/parallel/popen-gw0/test_pypi_url0/channeldata.json
[35mDEBUG   [0m conda_index.index:__init__.py:680 found subdirs {'noarch', 'linux-64'}
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:101 CondaIndexCache channel_root=/usr/share/miniconda/envs/test/conda-bld, subdir=linux-64 db_filename=/usr/share/miniconda/envs/test/conda-bld/linux-64/.cache/cache.db cache_is_brand_new=False
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:431 linux-64 listdir
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:450 linux-64 save fs state
[35mDEBUG   [0m conda_index.index:__init__.py:759 linux-64 find packages to extract
[35mDEBUG   [0m conda_index.index:__init__.py:771 linux-64 extract 0 packages
[32mINFO    [0m conda_index.index:__init__.py:801 linux-64 cached 0 B from 0 packages at 0 B/second
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:101 CondaIndexCache channel_root=/usr/share/miniconda/envs/test/conda-bld, subdir=noarch db_filename=/usr/share/miniconda/envs/test/conda-bld/noarch/.cache/cache.db cache_is_brand_new=False
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:431 noarch listdir
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:450 noarch save fs state
[35mDEBUG   [0m conda_index.index:__init__.py:759 noarch find packages to extract
[35mDEBUG   [0m conda_index.index:__init__.py:771 noarch extract 0 packages
[32mINFO    [0m conda_index.index:__init__.py:801 noarch cached 0 B from 0 packages at 0 B/second
[32mINFO    [0m conda_index.index:__init__.py:578 Completed linux-64
[32mINFO    [0m conda_index.index:__init__.py:578 Completed noarch
[35mDEBUG   [0m conda_index.index:__init__.py:680 found subdirs {'noarch', 'linux-64'}
[32mINFO    [0m conda_index.index:__init__.py:653 Channeldata subdir: linux-64
[35mDEBUG   [0m conda_index.index:__init__.py:654 linux-64 read repodata
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:101 CondaIndexCache channel_root=/usr/share/miniconda/envs/test/conda-bld, subdir=linux-64 db_filename=/usr/share/miniconda/envs/test/conda-bld/linux-64/.cache/cache.db cache_is_brand_new=False
[35mDEBUG   [0m conda_index.index:__init__.py:662 linux-64 channeldata finished
[32mINFO    [0m conda_index.index:__init__.py:653 Channeldata subdir: noarch
[35mDEBUG   [0m conda_index.index:__init__.py:654 noarch read repodata
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:101 CondaIndexCache channel_root=/usr/share/miniconda/envs/test/conda-bld, subdir=noarch db_filename=/usr/share/miniconda/envs/test/conda-bld/noarch/.cache/cache.db cache_is_brand_new=False
[35mDEBUG   [0m conda_index.index:__init__.py:662 noarch channeldata finished
[35mDEBUG   [0m conda_index.index:__init__.py:1155 _maybe_write /usr/share/miniconda/envs/test/conda-bld/index.html to /usr/share/miniconda/envs/test/conda-bld/index.html
[35mDEBUG   [0m conda_index.index:__init__.py:670 write channeldata
[35mDEBUG   [0m conda_index.index:__init__.py:1155 _maybe_write /usr/share/miniconda/envs/test/conda-bld/channeldata.json to /usr/share/miniconda/envs/test/conda-bld/channeldata.json